tools/unicode_properties_parse/format_width_estimate_intervals.cpp

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// The following code generates data for `_Width_estimate_intervals_v2` in <format>.
+
+#include <charconv>
+#include <cstdint>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <regex>
+#include <source_location>
+#include <string>
+#include <system_error>
+#include <vector>
+using namespace std;
+
+void verify(bool test, const char* msg, source_location loc = source_location::current()) {

```

Comment:
Hmm, is `const` needed here and in `range_u`'s constructors? (I think adding `const` in these places will make code less readable...)

Reply:
We prefer to use top-level `const` on value parameters, even though it's wordy, because it marginally improves safety and makes it more obvious when value parameters will be modified later. However, we aren't absolutely insistent upon this, especially in short functions, especially in test code.